Cargo.toml

```diff
@@ -38,15 +38,15 @@ uuid = { version = "^0.8", features = ["v4"], default-features = false }
 serde = { version = "^1.0", optional = true, default-features = false }
 serde_json = { version = "^1", features = ["std"], optional = true, default-features = false }
 valico = { version = "^3.2", optional = true, default-features = false }
-base64 = { version = "^0.12", optional = true, default-features = false }
+base64 = { version = "^0.13", optional = true, default-features = false }
 http = { version = "^0.2", optional = true, default-features = false }
-hyper = { version = "^0.13.1", optional = true, default-features = false }
-yup-oauth2 = { version = "4", optional = true, default-features = false }
-prost = { version = "0.6", optional = true, default-features = false }
+hyper = { version = "^0.14.4", optional = true, features = ["client", "stream"], default-features = false }
+yup-oauth2 = { version = "5", optional = true, features = ["hyper-rustls"], default-features = false }
```

Comment:
This should not depend on any SSL library, the API user provides the TLS-capable client.

Reply:
The reason this is here is that `yup-oauth2@5` has `hyper-rustls` as a default feature, and will not compile if at least one of the `hyper-tls` or `hyper-rustls` feature is enabled https://github.com/dermesser/yup-oauth2/compare/v4.1.3...v4.1.4

What is the preferred approach to dealing with this?

Reply:
Since `hyper-rustls` is the default, that's why I enabled it here, to preserve the `default-features = false` which I did to minimize the number of changes in the PR.